enso/metrics/basic_rationalized.py

```diff
@@ -12,14 +13,18 @@ class MacroRocAucRationalized(ClassificationMetric):
 
     def evaluate(self, ground_truth, result):
         """Return AUC metric."""
+
         ground_truth = [yi[1] for yi in ground_truth]
+
+        # for debugging 
+        print("DEBUGING INFO HERE!!!!")
+        print(Counter(ground_truth))
         classes = list(set(ground_truth) | set(result.columns))
         classwise_auc = {}
         binary_labels = labels_to_binary(ground_truth)
         default = np.zeros(shape=[len(ground_truth), 1], dtype=np.int32)
         binary_labels = np.hstack([binary_labels[column].values.reshape(-1, 1) if column in binary_labels else default for column in classes])
-        predicted_labels = np.hstack([result[column].values.reshape(-1, 1) if column in result.columns else default for column in classes])
-        return roc_auc_score(binary_labels, predicted_labels, average='macro')
+        predicted_labels = np.hstack([result[column].values.reshape(-1, 1) if column in result.columns else default for column in classes]) return roc_auc_score(binary_labels, predicted_labels, average='macro')
```

Comment:
This line looks like a syntax error to me?

Reply:
yeah, fixed that one.